src/main/registry.ts

```diff
@@ -92,6 +93,12 @@ export class Registry implements IRegistry, IDisposable {
 
     try {
       const defaultEnv = this._resolveEnvironmentSync(pythonPath);
+      if (defaultEnv.path !== getBundledPythonPath()) {
+        // If the default environment is not the bundled one, don't set it (this._defaultEnv).
+        // This is to prevent the default environment from being set to one of the user's 
+        // other environments. We should only be working with the bundled environment.
+        return;
+      }
```

Comment:
So if `this._defaultEnv` is not set, then we fall back to displaying the `install` button? 

If the answer to that question is `Yes`, then: 
One question here though is: It might be the case that the user 
1. has a different default path set
2. already has the bundled environment created 

In this case, we would not want to prompt them to click the `install` button. Instead, we would just want to select the bundled environment. 

To implement that would it make sense to just always set `this._defaultEnv` to the bundledPythonPath if it exists?